### DISCUSSION TITLE
Which property should I use when adding ownership verification in Google Search Console?

### DISCUSSION THREAD

**Discussion by george-gca** [alshedivat/al-folio#1184] [Q&A] [answered]
Just to be clear, when adding ownership verification of the site in Google Search Console, is it possible to use domain [property](https://support.google.com/webmasters/answer/34592?hl=en#types) if I deploy to `github.io`? Or only url prefix will work?

**Comment by sr-dash** [comment]
Hi @george-gca ! I just checked the ownership verification on my webpage. It looks like only URL profix is working for me not the domain property generated ID. 

Also, it'd be really helpful if a question in the FAQ can be posted regarding correct usage of google-site-verification tag in _config.yml file. 

The user has to generate a HTML tag following this discussion
[https://support.google.com/webmasters/answer/9008080?hl=en#meta_tag_verification&zippy=%2Chtml-tag](https://support.google.com/webmasters/answer/9008080?hl=en#meta_tag_verification&zippy=%2Chtml-tag)
 
and then copy the string against content="Google Search Console ID" and use it in the tag google-site-verification: Google Search Console ID after setting enable_google_verification: true to successfully verify the site ownership.

**Reply by george-gca** [reply]
Can you send a PR with this?

**Reply by sr-dash** [reply]
Since I created my copy of the repo as a template. It does not have the historical information to create a PR. Here is my recent push for the FAQ to my repo [https://github.com/sr-dash/sr-dash.github.io/blob/main/FAQ.md](https://github.com/sr-dash/sr-dash.github.io/blob/main/FAQ.md). I would love to create a PR with this but I need help with the syntax.

**Reply by george-gca** [reply]
To contribute to the repo you will need to make a fork of it, but you can keep it separate from your site. Unless you want me to pull the information from your repo and apply it here, but I prefer to have your contribution on the history rather than me copying it.

**Reply by sr-dash** [reply, marked as answer]
Thanks. I just created a new fork and opened a PR with this update on FAQ.
